config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml

```diff
@@ -6,11 +6,12 @@ presubmits:
         preset-kind-volume-mounts: "true"
         preset-cluster-api-provider-vsphere-e2e-config: "true"
         preset-cloud-provider-vsphere-e2e-config: "true"
-      always_run: false
-      optional: true
       decorate: true
+      decoration_config:
+        timeout: 1h
+      run_if_changed: 'images/capi/(Makefile|ansible\.cfg)|images/capi/ansible/.*|images/capi/scripts/ci-ova\.sh|images/capi/packer/(config|goss|ova)/.*|images/capi/hack/ensure-(ansible|packer|jq|azure-cli|goss).*'
```

Comment:
I think `azure-cli` goes in the other job.

Reply:
Thanks! fixed.